crates/ef-testing/src/bin/fetch_dump_katana.rs

```diff
@@ -19,39 +19,33 @@ struct WorkflowRun {
     head_branch: String,
 }
 
+/// Fetches the Katana dump from Kakarot-RPC artifacts
+/// On every PR and merge, thanks to a CI job in Kakarot-RPC, the state of a Starknet devnet
+/// with all of Kakarot Cairo smart contracts deployed is dumped in an artifact called 'dump-katana'
+/// Starting a Starknet local chain from checkpoint speeds up all our processes.
 fn main() -> Result<(), Box<dyn std::error::Error>> {
     dotenv().ok();
     let token =
         std::env::var("GITHUB_TOKEN").map_err(|_| eyre::eyre!("Missing GITHUB_TOKEN in .env"))?;
-    let url = "https://api.github.com/repos/sayajin-labs/kakarot-rpc/actions/artifacts";
+    let url = "https://api.github.com/repos/kkrt-labs/kakarot-rpc/actions/artifacts";
 
-    let client = reqwest::blocking::Client::builder();
-    let response: serde_json::Value = client
+    let client = reqwest::blocking::Client::builder()
```

Comment:
here refactoring a little because semantically it's only a client once it's built with .build(), otherwise it's still a builder